site/documentation/content/concepts/command-and-control.md

```diff
@@ -85,8 +89,12 @@ The following diagrams show the message flow for commands over the MQTT adapter:
 
 When a device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link specifying the following source address:
 
-* `control` for authenticated devices
-* `control/${tenant}/${device-id}` for unauthenticated devices
+* `command` for authenticated devices
+* `command/${tenant}/${device-id}` for unauthenticated devices
+
+{{% note %}}
+Previous versions of Hono used `control` instead of `command` as address prefix. Using the `control` prefix is still supported but deprecated. 
```

Comment:
I think we can omit this note from the concept page ... it is sufficient to have this in the API spec FMPOV

Reply:
Ok, I've removed it.